ToDoApp/ToDoApp/ToDoServiceImpl.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  ToDoServiceImpl.swift
+//  ToDoApp
+//
+//  Created by Trong Hieu Vuong on 2022-12-23.
+//
+
+import Foundation
+import FirebaseDatabase
+import FirebaseDatabaseSwift
+enum TaskError: Error {
+    case unavailableTask
+}
+
+struct TaskModel: Identifiable, Codable {
+    let id: UUID
```

Comment:
is this compiling?

Reply:
yes, it's compiling. And the app runs well
[Screen Recording 2022-12-23 at 4.24.51 PM.mov.zip](https://github.com/hieuvuong2310/ToDoApp/files/10297744/Screen.Recording.2022-12-23.at.4.24.51.PM.mov.zip)

Reply:
right, my fault. messed up with `let id: UUID = UUID()`. never mind :)